stko/molecular/networkx/network.py

```diff
@@ -0,0 +1,64 @@
+"""
+Network
+=======
+
+Class for defining a networkx graph from a molecule.
+
+"""
+
+import logging
+import networkx as nx
+
+logger = logging.getLogger(__name__)
+
+
+class Network:
+    """
+    Definition of a network of an stk.Molecule.
+
+    """
+
+    def __init__(self, molecule):
+        self._molecule = molecule
+        self._graph = self.get_graph()
+
+    def get_graph(self):
+        g = nx.Graph()
+        # Define by adding edges.
+        for bond in self._molecule.get_bonds():
+            g.add_edge(bond.get_atom1(), bond.get_atom2())
+
+        return g
+
+    def delete_bonds(self, atom_ids_to_disconnect):
+        atom_ids_to_disconnect = [
+            tuple(sorted(i)) for i in atom_ids_to_disconnect
+        ]
```

Comment:
overwriting the parameter is a little bit dangerous. easy to cause confusion alter as people might not notice the line.